workflow/github.md

```diff
@@ -1,2 +1,138 @@
 # Working on GitHub
 
+This document outlines the rules to contribute and to create issues to a project on the organisation [Canonical-websites](https://github.com/canonical-websites/)
+
+## Table of content
+
+* [How can I contribute?](#how-can-i-contribute-)
+  + [Report Bugs](#report-bugs)
+  + [Suggest Enhancements](#suggest-enhancements)
+  + [Work on an issue](#work-on-an-issue)
+* [Pull Requests](#pull-requests)
+  + [First contribution](#first-contribution)
+  + [Styleguides](#styleguides)
+    - [Commits messages](#commits-messages)
+    - [HTML](#html)
+    - [JS](#js)
+    - [Stylesheets](#stylesheets)
+    - [Python](#python)
+    - [Documentation](#documentation)
+  + [Merging process](#merging-process)
+* [Templates](#templates)
+  + [Issues template](#issues-template)
+  + [Pull-request template](#pull-request-template)
+
+## How can I contribute?
+
+In the Canonical web-sites organisation we work with Github. All types of work are tracked as GitHub issues. You will find in this section how you can contribute to this organisation.
+
+### Report Bugs
+
+Well done you have found a bug! Go to the corresponding project, it should be in the [Canonical websites organisation](https://github.com/canonical-websites/) and open an issue. These are the steps you need follow to add an issue:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Add labels:** see [Labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Suggest Enhancements
+
+If you think details could change on our websites you can suggest enhancements by creating a new issue. These are the steps you need follow to suggest enhancements:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Screenshots:** Don’t forget to add screenshots of suggestion designs to explain
+* **Add labels:** see [labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Work on an issue
+You are free to work on an issue if no one seems to work on it. Don’t forget to assign yourself to the issue. Once you finish the work open a [pull-request](#pull-Requests)
+
+## Pull Requests
+
+In this section you will find a description on how to create a pull-request for a Canonical Web-sites project. These are the steps to create a pull-request:
```

Comment:
I think "In this section you will find a description on how to create a pull-request for a Canonical Web-sites project" is superfluous. Such a short section doesn't need to tell you what it's going to say before it says it.

Also, could we link to an existing guide on filing pull-requests? Maybe [this one](https://help.github.com/articles/creating-a-pull-request/)?